test/firtool/firtool.fir

```diff
@@ -46,17 +46,19 @@ circuit test_mod : %[[{"class": "circt.testNT", "data": "a"}]]
 
 ; VERILOG-IR: sv.verbatim{{.*}}Standard header to adapt well known macros to our needs
 ; VERILOG-IR: sv.macro.def @INIT_RANDOM_PROLOG_
+  type V3 = UInt<1>[3]
+  type V2 = UInt<1>[2]
 
   module test_mod :
     input clock : Clock
     input a: UInt<1>
```

Comment:
It seems type alias for integers is not well supported in HW/Comb. There are still a lot of places where normal llvm casts are used. So i added type alias only for vectors.